src/sentry/utils/snuba.py

```diff
@@ -867,7 +872,7 @@ def get_cache_key(query: SnubaQuery) -> str:
 
 
 def _apply_cache_and_build_results(
-    snuba_param_list: Sequence[SnubaQueryBody],
+    snuba_param_list: Sequence[RequestQueryBody],
```

Comment:
we had other sentry files using this function, but I verified they all conform to the changed input spec